arviz/plots/distplot.py

```diff
@@ -88,14 +88,14 @@ def plot_dist(
         Ignored for 2D KDE
     rug_kwargs : dict
         Keywords passed to the rug plot. Ignored if rug=False or for 2D KDE
-        Use `space` keyword (float) to control the position of the rugplot. The larger this number
+        Use ``space`` keyword (float) to control the position of the rugplot. The larger this number
         the lower the rugplot.
     contour_kwargs : dict
         Keywords passed to the contourplot. Ignored for 1D KDE.
     contourf_kwargs : dict
-        Keywords passed to ax.contourf. Ignored for 1D KDE.
+        Keywords passed to :func:`ax.contourf`. Ignored for 1D KDE.
     pcolormesh_kwargs : dict
-        Keywords passed to ax.pcolormesh. Ignored for 1D KDE.
+        Keywords passed to :func:`ax.pcolormesh`. Ignored for 1D KDE.
     hist_kwargs : dict
         Keywords passed to the histogram.
```

Comment:
The 1st example passed the kwargs to another arviz function, and had the same behaviour for both bokeh and matplotlib, but this won't be the case often. This particular one however will be among the most complicated kwarg arguments to document. In plot_dist, most are passed to plot_kde as is so don't let it throw you back.

We start again like with rug_kwargs, taken and passed directly to the backend plotting function, see https://github.com/arviz-devs/arviz/blob/main/arviz/plots/distplot.py#L32, https://github.com/arviz-devs/arviz/blob/main/arviz/plots/distplot.py#L199 and https://github.com/arviz-devs/arviz/blob/main/arviz/plots/distplot.py#L211-L212).

### matplotlib
If you search for `hist_kwargs` in this file you'll see many ocurrences in https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/matplotlib/distplot.py#L66-L77. These are one example of what I was saying before. Ignore all that, you only have to follow the argument as it is passed downstream. 

Here is is passed to `_histplot_mpl_op` in https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/matplotlib/distplot.py#L84. This is an internal function that you'll see doesn't appear in the [docs](https://arviz-devs.github.io/arviz/api/plots.html), so the job is not finished yet.

[`_histplot_mpl_op` is defined below](https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/matplotlib/distplot.py#L125) and it passes the hist kwargs to the matplotlib hist function in https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/matplotlib/distplot.py#L155. 

### bokeh 
basically the same pattern as before but now passing them to the `_histplot_bokeh_op` at https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/bokeh/distplot.py#L79. Here this is eventually passed to the [`quad` function](https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/bokeh/distplot.py#L178). But there are some pop clauses: https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/bokeh/distplot.py#L127, https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/bokeh/distplot.py#L132, https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/bokeh/distplot.py#L136 and https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/bokeh/distplot.py#L138. Each of these pop things should have an explanatory sentence.

---
Proposal for the description of this argument:

```
Keyword arguments used to customize the histogram. Ignored when plotting a KDE. They are passed to
:meth:`matplotlib.axes.Axes.hist` if using matplotlib, or to :meth:`bokeh.plotting.Figure.quad` if
using bokeh. In bokeh case, the following extra keywords are also supported:

* ``color``: replaces the ``fill_color`` and ``line_color`` of the ``quad`` method
* ``bins``: taken from ``hist_kwargs`` and passed to :func:`numpy.histogram` instead
* ``density``: normalize histogram to represent a probability density function, Defaults to ``True``
* ``cumulative``: plot the cumulative counts. Defaults to ``False``
```